straxen/plugins/event_processing.py

```diff
@@ -659,12 +659,21 @@ class CorrectedAreas(strax.Plugin):
         default='cmt://rel_extraction_eff?version=ONLINE&run_id=plugin.run_id',
         help='Relative extraction efficiency for this run (allows for time dependence)')
 
+    # relative light yield
+    # defaults to no correction
+    rel_light_yield = straxen.URLConfig(
+        default='cmt://relative_light_yield?version=ONLINE&run_id=plugin.run_id',
```

Comment:
sorry @l-althueser and @a1exndr, what's the value for the future data? 1?

Reply:
Yes: https://github.com/XENONnT/private_nt_aux_files/pull/152/files#diff-c19121527752e93c5048c5104740d1470ffeb8b4152ce77c836ea3a8a4868e3fR354

Reply:
![image](https://user-images.githubusercontent.com/13698969/155969858-bec5d9f9-12cd-4a28-9c8f-0c1ae2b95d8f.png)